tests/integration/test_properties.py

```diff
@@ -215,15 +216,20 @@ def test_delete_property_by_id(client, auth_headers, test_database):
     assert response == 404
 
 
-def test_update_property_by_id(client, auth_headers, create_property, test_database):
+def test_update_property_by_id(client, empty_test_db, valid_header, create_property):
     test_property = create_property()
     new_property_address = "123 NE Flanders St"
-    test_property.address = new_property_address
 
-    """The property should have a new address"""
-    test_changed_property = client.get(
-        f"/api/properties/{test_property.id}", headers=auth_headers["admin"]
+    property_json = test_property.json()
+
+    property_json["address"] = new_property_address
+
+    response = client.put(
+        f"/api/properties/{test_property.id}",
+        json=property_json,
+        headers=valid_header,
     )
-    test_changed_property = json.loads(test_changed_property.data)
+    assert is_valid(response, 200)
```

Comment:
This seems good and all, but I don't think is_valid is retesting what the framework already tests. Testing the status code is great, but testing the headers are correct doesn't need to be done everywhere as that is done automatically by the tools we're using, and does not need to be retested in every endpoint test, especially since we're always returning json.